### ISSUE TITLE
Migrated server caused web browser to hang (Possibly in emoji.js ?)

### ISSUE THREAD

**Issue by lionpanthera** [mastodon/mastodon#24612]
### Steps to reproduce the problem

1. Migrate server (see #24611 )
2. Load web UI in Safari or Firefox

### Expected behaviour

No hang

### Actual behaviour

Hang

### Detailed description

After the pain of trying to migrate my server in #26411, I got everything working, but now my server is causing Safari (and Firefox) to hang.

The page loads, and the timeline displays, but then the browser hangs. Clicking anywhere results in nothing. Eventually the browser reports that the tab has stopped responding.

If you click "debug script" in Firefox after it hangs, it takes you to "emoji.js", but the highlighted line seems to be different each time. I don't know Javascript.

These errors appear in the console:

````
The resource at “https://mastodon.dgfp.club/packs/js/features/compose-dd226566ebaa619cf8ec.chunk.js” preloaded with link preload was not used within a few seconds. Make sure all attributes of the preload tag are set correctly.
The resource at “https://mastodon.dgfp.club/packs/js/features/home_timeline-6d1d4456eae4781f0256.chunk.js” preloaded with link preload was not used within a few seconds. Make sure all attributes of the preload tag are set correctly.
The resource at “https://mastodon.dgfp.club/packs/js/features/notifications-7c1404e72ebd1f736bfa.chunk.js” preloaded with link preload was not used within a few seconds. Make sure all attributes of the preload tag are set correctly.
````

(Additionally, people following from other mastodon servers say that my images aren't loading, but I can't begin to debug that with my web UI killing my browser.)

### Specifications

Mastodon 4.1.2
Ruby 3.2.2
Node.js 16.20.0

**Comment by lionpanthera** [comment]
This just... spontaneously fixed itself, for no apparent reason. Since I can no longer reproduce this, I'll close it until if/when it comes back. ¯\_(ツ)_/¯

**Comment by ClearlyClaire** [comment]
This was a bug, fixed by #24615

You don't appear to be running 4.1.2 but the development version, though.
